jenkins/deploy-clio-server.sh

```diff
@@ -115,6 +115,22 @@ compute_docker_tag() {
   echo ${base_version}-g${short_sha}-SNAP
 }
 
+# Build a "conventional" fully-qualified domain name from a hostname and environment.
+build_fqdn() {
+  echo ${1}.gotc-${ENV}.broadinstitute.org
+}
+
+# Get the "real" hostname for the Clio host at the given address (which might be a DNS).
+# Assumes that Clio hosts will be named according to the pattern:
+#  gotc-clio-{ENV}(\d{3})
+# i.e. gotc-clio-dev101, gotc-clio-prod-203
```

Comment:
in this example is there really a '-' in gotc-clio-prod-203?  Then I think the sed below won't work for that?